distribution/src/main/assembly/dist.xml

```diff
@@ -85,11 +90,17 @@
                 <exclude>**/zanata.sh</exclude>
                 <exclude>**/zanata.xml</exclude>
                 <exclude>src/main/scripts/**</exclude>
+                <exclude>hibernate-noorm-release-scripts/**</exclude>
 
                 <!-- actual files which should be ignored -->
+                <exclude>.travis.yml</exclude>
+                <exclude>travis/**</exclude>
                 <exclude>.git</exclude>
                 <exclude>.gitignore</exclude>
                 <exclude>**/target/**</exclude>
```

Comment:
Might be a bit agressive, isn't it? I hope we don't have a package named "target" somewhere...